src/lib/orbit_fit/orbit_fit.cpp

```diff
@@ -733,14 +743,12 @@ int orbit_fit(struct assist_ephem* ephem,
 	    lambda *= 2.0;
 	    
 	}
-	
-	//std::cout   << "chi2:\n" << chi2_d << std::endl;
-	//std::cout   << "rows: " << chi2.rows() << " cols: " << chi2.cols() << std::endl;      	
-	//std::cout   << "Cinv:\n" << C.inverse() << "\n";
 
+	/*
+	std::cout   << "chi2: " << chi2_d << std::endl;
 	std::cout << "lambda: " << lambda << std::endl;
-	std::cout << "chi2: " << chi2_d << std::endl;		
-	std::cout << "matrix dX\n" << dX << std::endl;
+	std::cout << "dX\n" << dX << std::endl;
+	*/
```

Comment:
Remove diagnostic comments

Reply:
we should have some runtime logging, but we should probably unify all logging into one object so that we can ensure that everything (from `python` and `C++` and elsewhere) ends up in a uniform place